commands/command_fetch.go

```diff
@@ -222,16 +222,15 @@ func fetchRecent(fetchconf lfs.FetchPruneConfig, alreadyFetchedRefs []*git.Ref,
 
 func fetchAll() bool {
 	pointers := scanAll()
-	Print("Fetching objects...")
+	Print("fetch: Fetching all references...")
 	return fetchAndReportToChan(pointers, nil, nil)
 }
 
 func scanAll() []*lfs.WrappedPointer {
 	// This could be a long process so use the chan version & report progress
-	Print("Scanning for all objects ever referenced...")
+	task := tasklog.NewSimpleTask()
```

Comment:
I'd add a `defer task.Complete()` for code consistency.

Reply:
Good call -- I picked this one up in 0661a8b.